tests/examples/shims/screen.lua

```diff
@@ -31,9 +31,12 @@ local function create_screen(args)
     local s = gears_obj()
     awesome._forward_class(s, screen)
 
-    s.data = {}
+    s._private = {}
     s.valid = true
 
+    -- Deprecated.
```

Comment:
mb also add info in which version it will be removed? (or better use gears deprecate so it will print the warning?)

Reply:
done